target/linux/ath79/image/tiny-tp-link.mk

```diff
@@ -55,6 +55,17 @@ define Device/tplink_tl-mr3420-v1
 endef
 TARGET_DEVICES += tplink_tl-mr3420-v1
 
+define Device/tplink_tl-mr3420-v2
+  $(Device/tplink-4mlzma)
+  ATH_SOC := ar9341
+  DEVICE_MODEL := TL-MR3420
+  DEVICE_VARIANT := v2
+  TPLINK_HWID := 0x34200002
+  DEVICE_PACKAGES := kmod-usb-core kmod-usb2 kmod-usb-ledtrig-usbport
```

Comment:
`kmod-usb-core` is an implicit dependency of `kmod-usb2`, so it can be dropped.

Reply:
Changes made